client/post-editor/editor-sidebar/index.jsx

```diff
@@ -49,6 +52,16 @@ export default React.createClass( {
 						selectedId={ this.props.post ? this.props.post.ID : null }
 					/>
 					: <div>
+					<EditorActionBar
+						isNew={ this.props.isNew }
+						onTrashingPost={ this.props.onTrashingPost }
+						onPrivatePublish={ this.props.onPublish }
+						post={ this.props.post }
+						savedPost={ this.props.savedPost }
+						site={ this.props.site }
+						type={ this.props.type }
+					/>
+					<EditorNotice { ...this.props.notice } />
```

Comment:
> The design image clearly says that the notice should appear in the tab where the user currently is (and be dismissed when he/she changes tab). I think that's the best experience, but I'll change it if there's agreement between the designers :)

At this stage I don't feel strongly either way: notice in both tabs vs switching on publish to the main tab and show it there. I'd probably go for the solution that has the least maintenance / code cost. ;)